logos/logos-ui/src/app/shared/components/search-input/search-input.scss

```
@@ -2,6 +2,7 @@
 
 .search-bar {
   @include glass-surface;
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)